packages/codec/tests/molecule.test.ts

```diff
@@ -316,7 +316,7 @@ test("nested type", (t) => {
     ["byteField", "arrayField", "structField", "fixedVec", "dynVec", "option"]
   );
 
-  const validInput: Parameters<typeof codec["pack"]>[0] = {
+  const validInput: Parameters<(typeof codec)["pack"]>[0] = {
```

Comment:
I find the prettier check is [failed](https://github.com/ckb-js/lumos/actions/runs/4324938395/jobs/7579516886#step:6:364) here, can you please help to prettier it again

Reply:
sorry for the inelegant push